system/jlib/jsecrets.cpp

```diff
@@ -31,9 +31,6 @@
 #if defined(__clang__) || defined(__GNUC__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wformat-nonliteral"
-
```

Comment:
was this deletion on purpose?

Reply:
@ghalliday - I've reverted that change (squashed in)